.circleci/config.yml

```diff
@@ -116,6 +116,9 @@ install_dep_180: &install_dep_180
   - run:
       name: Install Dependencies with torch 1.8.0 nightly
       command: |
+        # check if we have restored cache correctly, if so, just skip
+        python -c 'import torch; assert torch.__version__.split(".")[:2] != ["1", "8"]' || exit 0
```

Comment:
Should this be in the same PR? Or is this related?

Reply:
Yeah, CI run for this PR hit a failure related to venv installation. I decide to improve on that here. Ideally should split the PR, but it is very small change.